src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "white";
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_SIZE = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int figureType = random.nextInt(5);

```

Comment:
don't you get three same "random" figures with such an implementation? you should call random in a method scope. Final field is only set once. move 5 -> FIGURE_COUNT constant